vendors/espressif/boards/ports/ota_pal_for_aws/ota_pal.c

```diff
@@ -80,7 +91,7 @@ static esp_ota_context_t ota_ctx;
 static const char * TAG = "ota_pal";
 
 /* Specify the OTA signature algorithm we support on this platform. */
-const char cOTA_JSON_FileSignatureKey[ OTA_FILE_SIG_KEY_STR_MAX_LENGTH ] = "sig-sha256-ecdsa";
+const char OTA_JsonFileSignatureKey[ OTA_FILE_SIG_KEY_STR_MAX_LENGTH ] = "sig-sha256-ecdsa";
```

Comment:
Is there any documentation that needs to be updated alongside changing this variable name?